FBSnapshotTestCase/FBSnapshotTestController.m

```diff
@@ -226,6 +228,15 @@ - (NSString *)_fileNameForSelector:(SEL)selector
   if (0 < identifier.length) {
     fileName = [fileName stringByAppendingFormat:@"_%@", identifier];
   }
+  
+  if (self.isDeviceAgnostic) {
+    UIDevice *device = [UIDevice currentDevice];
+    NSString *model  = [device.model stringByReplacingOccurrencesOfString:@" " withString:@"_"];
+    NSString *os     = [device.systemVersion stringByReplacingOccurrencesOfString:@"." withString:@"_"];
+
+    fileName         = [NSString stringWithFormat:@"%@_%@%@", fileName, model, os];
```

Comment:
nit: We don't tend to align on the `=`; I'd prefer if we stuck to a single space before and after the `=` here.